### PR TITLE
Added linter fixes

### PR DESCRIPTION
**Pull request by sawood14012**

### CODE REVIEW

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -20,7 +20,11 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with: 
           version: v1.29
-          working-directory: gomanifest   
+          working-directory: gomanifest
+          #args: --enable-all
```
Comment (deepak1725) [inline .github/workflows/ci-build.yml:24]: Can you remove this commented part.

File: .github/workflows/ci-build.yml [hunk 2]
```diff
@@ -20,7 +20,11 @@ jobs:
...
+      - name: Linter
```
Comment (deepak1725) [inline .github/workflows/ci-build.yml:25]: We are adding Separate linter, in other words, two linters will exists parallelly, Why are we nesting one in another..?

**Review by deepak1725 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-build.yml (hunk 1 above)
Comment (sawood14012) [reply .github/workflows/ci-build.yml]: done

File: .github/workflows/ci-build.yml (hunk 2 above)
Comment (sawood14012) [reply .github/workflows/ci-build.yml:24]: do you want me to have it as a seprate job/step ?

Comment (sawood14012) [reply .github/workflows/ci-build.yml]: done

**Review by deepak1725 — APPROVED** [review]
lgtm

**Pull request merged**